ctapipe/image/extractor.py

```diff
@@ -347,6 +351,11 @@ class FixedWindowSum(ImageExtractor):
         "(peak_index - shift)",
     ).tag(config=True)
 
+    apply_integration_correction = BoolTelescopeParameter(
+        default_value=True,
+        help="Weither or not to apply the integration window correction",
```

Comment:
typo "Weither" -> "Whether"
Simply "Apply the integration window correction" would suffice